pkgs/development/beam-modules/rebar3-release.nix

```diff
@@ -44,6 +44,8 @@ let
     name = "${name}-${version}";
     inherit version;
 
+    nativeBuildInputs = [ gnused ];
+
     buildInputs = buildInputs ++ [ erlang rebar3 openssl ] ++ beamDeps;
```

Comment:
if erlang is defined as a buildInput it will end up in the resulting derivation right?
Maybe you want to shit this to a nativebuildinput ?
rebar3 as well?

Reply:
Hmm, tbh I don't fully understand the difference between `nativeBuildInputs` and `buildInputs`. I guess it's only relevant for cross-compiling?

Reply:
My understanding is that nativeBuildInputs is to signal that this dependency will only be needed during build. Whereas buildInputs are things that could be needed at runtime as well.
https://discourse.nixos.org/t/use-buildinputs-or-nativebuildinputs-for-nix-shell/8464